spec/models/micropost_spec.rb

```diff
@@ -1,11 +1,17 @@
 require 'rails_helper'
 
 RSpec.describe Micropost, type: :model do
-  before(:each) do
-    @user_mike = create(:user_mike)
-    @micropost = @user_mike.microposts.build(content: 'Lorem ipsum')
-  end
+  let(:user_mike) { create(:user_mike) }
+  let(:micropost) { user_mike.microposts.create(content: 'Lorem ipsum') }
+
   it 'micropost shoud valid' do
-    expect(@micropost.valid?).to eq(true)
+    expect(micropost.valid?).to eq(true)
+  end
+
+  it 'micropost shoud f1 should have 1 reply' do
```

Comment:
sau `it` sẽ là 1 động từ như các chỗ khác em viết
it "should"
it "expect"
it "change"
it "delete"
.....